### DISCUSSION TITLE
smtp error

### DISCUSSION THREAD

**Discussion by reetikpant** [postalserver/postal#2065] [Help with using Postal]
everything is working fine, i have checked by logging in on browser, but when i'm using creds on thrid party software to connect my smtp, its showing "A connection attempt failed" what to do?

**Comment by willpower232** [comment]
You can verify that your server has port 25 open by using telnet from another computer or otherwise have a look at the logs when you're trying to connect with `postal logs smtp`

**Reply by reetikpant** [reply]
![88](https://user-images.githubusercontent.com/97339805/178156587-e0d0aa41-2177-4b84-a1fc-88fcc14038ae.png)


here's postal logs

**Reply by reetikpant** [reply]
![89](https://user-images.githubusercontent.com/97339805/178156788-5745ff62-7a9e-4d31-87e6-5f8ee4209f35.png)

I'm guessing its connecting to port 25, that means port 25 is not blocked

**Reply by willpower232** [reply]
Looks like everything is working then, you should have a look at the logs after you do a test connection to see what it is doing.

**Comment by systmfr** [comment]
Haha we have the same thing going on, let me know if you figure something out too

**Reply by reetikpant** [reply]
ok sure, you too if u find first
